src/main/java/net/dv8tion/jda/core/requests/ratelimit/ClientRateLimiter.java

```diff
@@ -55,8 +52,6 @@ public Long getRateLimit(Route.CompiledRoute route)
     @Override
     protected void queueRequest(Request request)
     {
-        if (isShutdown)
-            throw new RejectedExecutionException("Cannot queue a request after shutdown");
```

Comment:
I removed it because `Requester#request()` already checks for it and it's unnecessary to do it twice:
```java
public <T> void request(Request<T> apiRequest)
{
    if (rateLimiter.isShutdown) 
        throw new IllegalStateException("The Requester has been shutdown! No new requests can be requested!");
    if (apiRequest.shouldQueue())
        rateLimiter.queueRequest(apiRequest);
    else
        execute(apiRequest, true);
}
```
This is the only place `RateLimiter#queueRequest()` is called from.